upgrade-instructions.md

```diff
@@ -256,9 +256,17 @@ Concerned products: Guice distributed James (rabbitMQ)
 
 The distributed James project (relying on Guice, Cassandra, ElasticSearch, RabbitMQ and optionally Swift) benefits from a new distributed task mananger.
 
-In order to enforce task sequential processing at the cluster level, we rely on a single active consumer, which is a feature introduced in RabbitMQ 3.8.
+In order to enforce task sequential processing at the cluster level, we rely on a single active consumer, which is a feature introduced in RabbitMQ 3.8.1.
 
-Users of distributed James product thus need to upgrade their RabbitMQ server to be at least version 3.8.
+Users of distributed James product thus need to upgrade their RabbitMQ server to be at least version 3.8.1.
```

Comment:
so 3.8.0 here?

Reply:
We do not test it on RQ 3.8.0, so, recommending an untested version looks dangerous.